lib/vector-buffers/Cargo.toml

```diff
@@ -14,11 +14,13 @@ bytes = { version = "1.5.0", default-features = false }
 crc32fast = { version = "1.3.2", default-features = false }
 crossbeam-queue = { version = "0.3.8", default-features = false, features = ["std"] }
 crossbeam-utils = { version = "0.8.16", default-features = false }
+derivative = { version = "2.2.0", default-features =  false }
 fslock = { version = "0.2.1", default-features = false, features = ["std"] }
 futures = { version = "0.3.29", default-features = false, features = ["std"] }
 memmap2 = { version = "0.9.0", default-features = false }
 metrics = "0.21.1"
 num-traits = { version = "0.2.17", default-features = false }
+paste = "1.0.14"
```

Comment:
Why `paste`? I don't see where it's being used.

Reply:
I needed to add it to use for the `registered_event` macro in this crate.

Reply:
A-ha. A problem to fix another day. 👍🏻